gradient/src/main/kotlin/br/com/concrete/yosef/gradient/property/ColorArrayCommand.kt

```diff
@@ -0,0 +1,80 @@
+package br.com.concrete.yosef.gradient.property
+
+import android.graphics.Color
+import android.graphics.drawable.GradientDrawable
+import android.text.TextUtils
+import android.view.View
+import br.com.concrete.yosef.api.property.DynamicPropertyCommand
+import br.com.concrete.yosef.canBeConvertedToInt
+import br.com.concrete.yosef.entity.DynamicProperty
+
+class ColorArrayCommand : DynamicPropertyCommand {
+
+    companion object {
+        const val COLORS = "colors"
+    }
+
+    override fun apply(view: View, dynamicProperty: DynamicProperty) {
+
+        if (dynamicProperty.type == "colors") {
+            val value = dynamicProperty.value
+
+            if (valueCanBeParsed(value)) {
+                val parsedColors = ArrayList<Int>()
+                val colors = value.split(",")
+                colors.forEach {
+                    val rgba = it.trim().split(".")
+                    val parsedColor = Color.argb(
```

Comment:
What do you think of destructuring (https://kotlinlang.org/docs/reference/multi-declarations.html) here?